dinner/readme.md

```diff
@@ -4,13 +4,18 @@
 
 ## Time to prepare: 1 hour
 
-## Ingredients: Cheese, Mozarella, Homemade Tomato Sauce, Flour, Water, Salt, Oregno, Olive Oil
+## Ingredients: 
+Mozarella, Homemade Tomato Sauce, Flour, Water, Salt, Oregno, Olive Oil
 
 
-## Equipment needed: Pizza Stone, Oven, Bowls, Taste Buds
+## Equipment needed: 
+Pizza Stone, Oven, Bowls, Taste Buds
 
 
-## Steps for preparation: 1) Make the Dough 2) Put the suace on the dough 3) Put cheese on the dough 4) Put it in the oven
+## Steps for preparation: 
+1) Make the Dough
+2)   Put the suace on the dough
```

Comment:
there seems to be an extra space here for no reason. Why would you do that 🥇

Reply:
sharp eyes!

Reply:
True i have to fix that.